x-pack/plugins/endpoint/server/routes/metadata/query_builders.ts

```diff
@@ -4,14 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { KibanaRequest } from 'kibana/server';
-import { EndpointAppConstants } from '../../../common/types';
+import { EndpointAppConstants, MetadataIndexGetBodyResult } from '../../../common/types';
 import { EndpointAppContext } from '../../types';
 import { esKuery } from '../../../../../../src/plugins/data/server';
 
+/**
+ * TODO, add comment to exported function.
+ */
 export const kibanaRequestToMetadataListESQuery = async (
-  request: KibanaRequest<any, any, any>,
+  request: KibanaRequest<unknown, unknown, MetadataIndexGetBodyResult>,
```

Comment:
Use the type based on the schema instead of `any`, this will help keep the client and server in sync, as they both use the same type. If the server changes, the type will have to change and then any invalid code in the UI will be detected.